### ISSUE TITLE
Support for teams?

### ISSUE THREAD

**Issue by fpedroza** [FormidableLabs/pull-report#25]
Does this support organization teams?

**Comment by fpedroza** [comment]
I don't believe this noted restriction would be the issue, but perhaps related?

> pull-report retrieves at most 100 pull requests/issues for any repo.

**Comment by ryan-roemer** [comment]
Hi!

For teams it doesn't directly support them. The query level is `org` and then pull-report manually filters things down to just a user list if supplied by the user.

For the limit, it's set here https://github.com/FormidableLabs/pull-report/blob/master/pull-report.js#L60-L71 . The needed change would be to use `@octokit/rest` (the new name for the `github` module) to paginate through results passing the page token until all results are done.  _Or_ maybe something more efficient could be done with the Search query API instead of the direct issues or pulls API.

See, e.g.:

- `page` parameter in https://octokit.github.io/rest.js/#octokit-routes-pulls
- https://developer.github.com/v3/pulls/#list-pull-requests (see `next` and `last` response for the raw API requests).

All in all, though, this library is ancient. Given how simple it is, it might be actually easier for a ground-up rewrite using `@octokit/rest` and the modern libs... Good luck!
